test/e2e/marketplace_test.go

```diff
@@ -1,41 +1,67 @@
 package e2e
 
 import (
-	"context"
 	"fmt"
 	"testing"
 
+	olm "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
 	"github.com/operator-framework/operator-marketplace/pkg/apis"
 	marketplace "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
-
-	olm "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	"github.com/operator-framework/operator-marketplace/test/helpers"
+	testsuites "github.com/operator-framework/operator-marketplace/test/testsuites"
```

Comment:
Isn't this alias unnecessary since it is the same string as the package name?

Reply:
Yep, I will remove the alias.

Reply:
Did the same for helpers in the test suite files.